### PR TITLE
Add example of abs on a positive number

### PR DESCRIPTION
**Pull request by henbestaj**
### Description

In \docs\content\python\concepts\built-in-functions\terms\abs changes abs.md to include an example of the function working on a positive number so that those who may not know what an absolute value is don't think that the abs python function simply reverses the sign of the input.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@henbestaj, this looks fine as is, and I'm passing it on for a second review.

File: content/python/concepts/built-in-functions/terms/abs/abs.md
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/abs/abs.md:45]:
```suggestion
The following example returns the absolute value from each element in the `numbers` list:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hi @henbestaj, thank you for flagging this! I was wondering if we could make a few more changes to this file before merging? Here are some suggestions:

- In the `Description` on line # 3, let's rewrite to the following:
  > Description: 'Returns the absolute value of a numeric argument.'
- Under `CatalogContent` (lines # 11 - 14), let's remove `'paths/computer-science'` from this list (we're trying to keep to just two links under this metadata item).
- On line # 17, let's rewrite the sentence like this:
  > The \*\*`abs()`\*\* function returns the absolute value of a numeric argument.
- Line # 21, let's change the code block signature to \```pseudo.
- Below line # 23, let's add 1-3 sentences explaining what `n` is in relation to the ` abs()` function.
- On line # 25, let's change to "## Example"
  - In this example code, let's set up two variables; one positive number and one negative number; and `print()` the two.

**Comment by henbestaj** [comment]
I've implemented all of the suggested changes into the abs() documentation.

File: content/python/concepts/built-in-functions/terms/abs/abs.md
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/abs/abs.md:24]:
```suggestion
The return value will be the absolute value of the `n` parameter, which is of type `int` or `float`.

The absolute value of `n` will be its distance from zero regardless of its direction (i.e., whether it is positive or negative). Since the absolute value is never negative, a positive value will remain unchanged and a negative value will have its negative sign removed
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@henbestaj Just one more suggestion 😄

**Comment by henbestaj** [comment]
How's this look?

**Review by Dusch4593 — APPROVED** [review]
@henbestaj LGTM! I made a few small edits in fa7890aef965862032296658e84916bcf46f46b6 that I wanted to note here:

- I **bolded** the first use of `abs()` on line # 16.
- Rephrased the example text on line # 30.
- Extracted the `# Output: ` comments into a separate \```shell snippet.

**Comment by Dusch4593** [comment]
Thank you so so much @henbestaj! 🥳 🎊 🎉 

Your changes should now be live: https://www.codecademy.com/resources/docs/python/built-in-functions/abs

Screen Shot 2022-10-30 at 9 46 05 PM

**Pull request merged**
